api/src/lib.rs

```diff
@@ -2600,6 +2600,126 @@ ffi_export!(
     }
 });
 
+ffi_export!(
+    fn Java_com_horizen_commitmenttree_CommitmentTree_nativeGetFwtLeaves(
+    _env: JNIEnv,
+    _commitment_tree: JObject,
+    _sc_id: jbyteArray
+) -> jobject
+{
+    let sc_id = {
```

Comment:
You can use the "deserialize_to_jobject" function to save a little bit of code lines and duplication